roles/kube-gateway/files/nginx.yaml

```diff
@@ -0,0 +1,10 @@
+init_config:
```

Comment:
I am not really sure how building this file into the farva docker container would help simplify things because this file has to live in the gateway pod on the container and then its copied onto the dd-agent container.